README.md

```diff
@@ -2,33 +2,55 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/7243b915eb214102be03f7c7b4dd2681)](https://app.codacy.com/gh/physarumAdv/Physarum_webGL?utm_source=github.com&utm_medium=referral&utm_content=physarumAdv/Physarum_webGL&utm_campaign=Badge_Grade_Settings)
 
-Visualization written on JS library THREE.js of a clever mould Physarum Polycephalum
-
-You can view this project [here](http://pavtiger.silaeder.ru/physarum)
-
-## Physarum Polycephalum 
-In nature is a mould that minimizes energy on
-growth process and food transfer. This way it builds the shortes and most optimal ways between points
-
-![Screenshot](readme_images/physarum.gif)
-
-## Cube
-Now growth is visualized at a single cube but mesh can be imported as a .gltf 3D format
-![Screenshot](readme_images/cube.jpg)
-
-## Growth
-![Screenshot](readme_images/mold.jpg)
-
-## Change color of the mold
-Using a slider at the top-right side of the screen you can change the color of the mold to anything you want while simulation is running
-![Screenshot](readme_images/color.jpg)
+This is a visualizer for Physarum Polycephalum's growth written in JavaScript using THREE.js and Node.js libraries.
+We recommend using [this](https://github.com/physarumAdv/minds_crawl) simulator. <br/>
+You can also visit this project on a website: [pavTiger.silaeder.ru](http://pavtiger.silaeder.ru)
+
+## Visualizer
+There are two visualizer modes:
+* **Offline**: Show a precalculated growth simulation
+
+* **Online**: If a simulator is working then it will show live growth. <br/>
+     ![](readme_images/interface_small.png)
+
+## Physarum Polycephalum
+In nature, this is a mold that minimizes energy on
+growth process and food transfer. This way it builds the shortest, and most optimal paths between points.
+<br/> ![](readme_images/physarum.gif) <br/>
+In this gif a group of Japanese scientists made an experiment when they grew Physarum on the map of Japan and it successfully reconstructed Japan's road system between cities.
+
+## Polyhedron
+Currently, we support all convex polyhedrons, but in future we plan to support any .obj 3D model. <br/>
+![](readme_images/poly.jpg)
+
+## Controls
+Using controls you can change a variety of parameters like mold's size and color.
+Moreover, you can control the speed, timeline and a few more parameters in Offline mode <br/>
+![](readme_images/panel.png)
+
+## Render mode
+As you can notice we have a special tab named "Render".
+The main purpose is to render a high quality video (.mp4) of the growth, but it is mostly made for us, however other users can still use it.
+When you tick this, website starts sending photos of the current canvas to the server, and later they get merged into a video.
+This way all renderings are saved on the server and browser doesn't have to merge them.
```

Comment:
Not a readme issue, just asking about the feature described here (you or I can start a new issue if you think it is worthy). Are you sure it is good to shift this work to the server? Won't it be loaded to much? Maybe the browser should do this?